scripts/README.md

```diff
@@ -0,0 +1,21 @@
+<h2> General-purpose scripts for Reactome's release </h2>
+
+This repository contains scripts that are used by multiple release steps.
+
+
+<h3>updateDatabase.sh</h3>
+
+<b>Purpose</b>: Create or replace a mySQL database with a mysql dump file.
```

Comment:
Small nitpick - I think the capitalization is usually MySQL -- could that be updated through the PR?